nixos/modules/services/misc/ntfy-sh.nix

```diff
@@ -59,6 +59,10 @@ in
         systemPackages = [ cfg.package ];
       };
 
+      services.uptime-kuma.settings = {
+        auth-file = "/var/lib/ntfy-sh/user.db";
```

Comment:
`lib.mkDefault` would be good

Reply:
Sure

Reply:
Also, is this needed for this PR? (which should only creates the data folder?)

Reply:
I feel it is a nice default to have for this module. It makes it usable out of the box while still leaving the chance for the user to change this config.
Creating the data folder is not actually needed to run ntfy-sh, it is needed to create users to I feel these two issues are bundled.

Reply:
(I'll also add that the auth-file option in ntfy has no default so if unset the user creation will not work)

Reply:
Can you rename the commit to match the PR title?

Reply:
Done :)